guides/introduction/plugins.md

```diff
@@ -1,8 +1,5 @@
 # Plugins
 
-- [still_snowpack](https://github.com/still-ex/still_snowpack) adds
```

Comment:
Can we add a deprecation notice to the project and archive it?

Reply:
Donest